Framework/Algorithms/src/CreateWorkspace.cpp

```diff
@@ -232,9 +254,6 @@ void CreateWorkspace::exec() {
     }
   }
 
-  // Set distribution flag
-  outputWS->setDistribution(getProperty("Distribution"));
-
   // Set Y Unit label
```

Comment:
The (unchaged) code for handling the vertical axis below is probably not correct in the MPI case, unless it is simply a spectrum axis. I think this requires some more investigation amongst more algorithms.